src/parser/expression/window_expression.cpp

```diff
@@ -234,6 +242,7 @@ void WindowExpression::Serialize(Serializer &serializer) {
 	serializer.WriteOptional(end_expr);
 	serializer.WriteOptional(offset_expr);
 	serializer.WriteOptional(default_expr);
+	serializer.Write<bool>(ignore_nulls);
```

Comment:
Adding/removing something to `ParsedExpression` serialization requires a bump to the storage version.

The reason for this is that any expression can be part of a macro, and macro's are serialized into the db file by serializing the `ParsedExpression`s because they need to be persistent. Anyone with window expression macro's may run into an error.

We (read 'I') should probably add a test that tries serializing a macro with each possible expression type so we can catch these things.

Reply:
Ah, I was wondering.  I'll see if I can add something to the test.

Reply:
Version bumped, test added.